staffSchedules/models.py

```diff
@@ -1,14 +0,0 @@
-from django.db import models
-
-from employees.models import Employee
-
-
-# Create your models here.
-class staffSchedule(models.Model):
-    employee = models.ForeignKey(Employee, on_delete=models.CASCADE)
-    start_time = models.TimeField()
-    end_time = models.TimeField()
-    days = models.IntegerField()
-
-    def __str__(self):
-        return str(self.employee)
```

Comment:
str(self.employee) 
str function is an extra call here. if we fix the employee class, add add the __str__ function there, it would be needed here. What do you think?